src/Module/Util/VaInfo.php

```diff
@@ -401,7 +402,7 @@ public static function get_tag_type($results, $configKey = 'metadata_order')
             'getid3_tag_order' => static::getConfigContainer()->get(ConfigurationKeyEnum::GETID3_TAG_ORDER)
```

Comment:
does static::getConfigContainer()->get return arrays or strings?

Reply:
@lachlan-00 Should be the same as AmpConfig::get.

Reply:
both return arrays:
example for metadata_oder
$order : array (
  0 => 'getid3',
  1 => 'musicbrainz',
  2 => 'theaudiodb',
  3 => 'filename',
)

Reply:
thank you for confirming